js/filters/localeOrderBy_filter.js

```diff
@@ -29,6 +39,10 @@ angular.module('contactsApp')
 
 			if (angular.isArray(valueA)) {
 				if (valueA[0] === valueB[0]) {
```

Comment:
We should probably also loop the array instead of supposing there is 2 elements in each one :)